templates/workspaces/base/terraform/airlock/storage_accounts.tf

```diff
@@ -230,6 +229,31 @@ resource "azurerm_private_endpoint" "export_inprogress_pe" {
   }
 }
 
+# Enable Airlock Malware Scanning on Core TRE for Export In-Progress
+resource "azapi_resource_action" "enable_defender_for_storage_export" {
+  count       = var.enable_malware_scanning ? 1 : 0
+  type        = "Microsoft.Security/defenderForStorageSettings@2022-12-01-preview"
+  resource_id = "${azurerm_storage_account.sa_export_inprogress.id}/providers/Microsoft.Security/defenderForStorageSettings/current"
+  method      = "PUT"
+
+  body = jsonencode({
+    properties = {
+      isEnabled = true
+      malwareScanning = {
+        onUpload = {
+          isEnabled     = true
+          capGBPerMonth = 5000
```

Comment:
does this value need to be a parameter , or is `5000` an agreed value?

Reply:
TBH this is copied straight from core where it was already in use.